### PR TITLE
added new util classes and enhanced the row-center class

### PR DESCRIPTION
**Pull request by KhaledElAnsari**
### Description

add some `text-align` util classes that can be used anywhere and will cut some styling effort

also improved the `row-center` class for better column alignment

in the second commit I added `.stack` class to force `flex-direction: row;` for the mobile

in the third commit I added `.fluid` class to make the `container` width full screen

### Code sample

new classes are:
    - `.text-left`
    - `.text-right`
    - `.text-center`
    - `.stack`
    - `.fluid`

properties used in `.row-center`:
    - `justify-content: center`
    - `text-align: center`

### CODE REVIEW

**Comment by KhaledElAnsari** [comment]
Guys I made different commits after this pull request was made and included them in the description

**Comment by KhaledElAnsari** [comment]
@cjpatoilo hey man, I've seen a lot of PRs passing the tests and getting flags (including this one) so when you will start accepting them?

File: src/_Grid.sass [hunk 1]
```diff
@@ -4,155 +4,163 @@
...
-	margin: 0 auto
-	max-width: 112.0rem
-	padding: 0 2.0rem
-	position: relative
-	width: 100%
+    margin: 0 auto
+    max-width: 112.0rem
+    padding: 0 2.0rem
+    position: relative
+    width: 100%
```
Comment (cjpatoilo) [inline src/_Grid.sass]: There is something wrong with these lines. There is only change is the spacing.

File: src/_Grid.sass [hunk 2]
```diff
@@ -4,155 +4,163 @@
...
 // up the height of the tallest .column in the same .row
 .row
-	display: flex
-	flex-direction: column
-	padding: 0
-	width: 100%
+    display: flex
+    flex-direction: column
+    padding: 0
+    width: 100%
```
Comment (cjpatoilo) [inline src/_Grid.sass]: There is something wrong with these lines. There is only change is the spacing.

File: src/_Grid.sass [hunk 3]
```diff
@@ -4,155 +4,163 @@
...
+
+    &.row-no-padding
+        padding: 0
 
-	&.row-no-padding
-		padding: 0
```
Comment (cjpatoilo) [inline src/_Grid.sass]: There is something wrong with these lines. There is only change is the spacing.

File: src/_Utility.sass [hunk 4]
```diff
@@ -5,14 +5,23 @@
...
 .clearfix
 
-	&:after
-		clear: both
-		content: ' ' // The space content is one way to avoid an Opera bug.
-		display: table
+    &:after
+        clear: both
+        content: ' ' // The space content is one way to avoid an Opera bug.
+        display: table
```
Comment (cjpatoilo) [inline src/_Utility.sass]: There is something wrong with these lines. There is only change is the spacing.

File: src/_Grid.sass (hunk 1 above)
Comment (cjpatoilo) [reply src/_Grid.sass:11]: @KhaledElAnsari look here!

File: src/_Grid.sass (hunk 2 above)
Comment (cjpatoilo) [reply src/_Grid.sass:25]: @KhaledElAnsari look here

**Comment by cjpatoilo** [comment]
@KhaledElAnsari For now, I will close this PR. Let me know if you understand the questions.

**Pull request closed without merging**